fares-data-build-tool/fdbt-site
# feat: Added describeSalesOfferPackage page

# Description

-   This PR adds an extra page to the sales offer package user flow, which allows a user to provide a name and description for their sales offer package.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that the new page works in conjunction with the other pages on the sales offer package user flow. Also validate that the page handles error cases as would be expected.

# Type of change


# Checklist:


<img width="1680" alt="Screenshot 2020-07-21 at 11 41 19" src="https://user-images.githubusercontent.com/55277550/88045886-24477200-cb47-11ea-814b-51ed00e4defc.png">
